pom.xml

```diff
@@ -5,8 +5,8 @@
 
     <parent>
         <groupId>io.smallrye</groupId>
-        <artifactId>smallrye-parent</artifactId>
-        <version>28</version>
+        <artifactId>smallrye-build-parent</artifactId>
+        <version>29-SNAPSHOT</version>
```

Comment:
Will need to be changed before the merge.

Reply:
Version yes